tests/integration/__init__.py

```diff
@@ -979,30 +980,30 @@ def transplant_configs(cls, transport='zeromq'):
             running_tests_user = win32api.GetUserName()
         else:
             running_tests_user = pwd.getpwuid(os.getuid()).pw_name
-        master_opts = salt.config._read_conf_file(os.path.join(CONF_DIR, 'master'))
```

Comment:
If you can't have the user setting on windows, make this setting check for that, but don't remove it.